app/classifier/tasks/survey/index.spec.js

```diff
@@ -1,72 +1,162 @@
       choice = wrapper.find(Choice);
-      assert.equal(choice.props().annotationValue.answers.ho, undefined);
-      assert.equal(choice.instance().state.answers.ho, undefined);
+      expect(choice.props().annotationValue.answers.ho, undefined).to.be.equal;
+      expect(choice.instance().state.answers.ho, undefined).to.be.equal;
+    });
+  });
+
+  describe('functions', function() {
+    let clock;
+    let wrapper = shallow(
+      <SurveyTask
+        annotation={annotation}
+        onChange={onChangeSpy}
+        translation={task}
+        task={task}
+      />
+    );
+
+    before(function () {
+      const date = new Date();
+      clock = sinon.useFakeTimers(date.getTime());
+    });
+
+    it('should correctly call handleAnnotation', function() {
+      wrapper.instance().handleAnnotation(selection.choice, selection.answers);
+      expect(onChangeSpy.calledOnce).to.be.true;
+      clock.tick();
+      const returnValues = onChangeSpy.returnValues[0];
+      expect(returnValues, annotation).to.be.equal;
+    });
+
+    it('should correctly remove an annotation on handleRemove', function() {
+      const currentLength = wrapper.instance().props.annotation.value.length;
+      wrapper.instance().handleRemove('ar');
+      const newLength = wrapper.instance().props.annotation.value.length;
+      expect(currentLength, newLength).to.not.equal;
+      expect(newLength).to.equal(0);
+    });
+
+    it('should call handleChoice', function() {
+      wrapper.instance().handleChoice('aa');
+      expect(wrapper.instance().state.selectedChoiceID, 'aa').to.be.equal;
+    });
+
+    describe('handleFilter', function () {
+      it('should add the correct filter', function() {
+        const expectedFilter = { aa: 'bb' };
+        wrapper.instance().handleFilter('aa', 'bb');
+        clock.tick();
+        expect(wrapper.state().filters, expectedFilter).to.be.equal;
+      });
+
+      it('should remove the correct filter', function () {
+        wrapper.instance().handleFilter('aa', undefined);
+        clock.tick();
+        expect(wrapper.state().filters, {}).to.be.equal;
+      });
+    });
+
+    after(function () {
+      clock.restore();
+    });
+  });
+
+  describe('static methods', function () {
+    it('should correctly return isAnnotationComplete', function() {
+      const isComplete = SurveyTask.isAnnotationComplete(task, annotation);
+      expect(isComplete).to.be.a('boolean');
+      expect(isComplete).to.be.true;
+    });
+
+    it('should correctly return a default annotation', function() {
+      const defaultAnnotation = SurveyTask.getDefaultAnnotation();
+      expect(defaultAnnotation).to.be.an('object');
+      expect(defaultAnnotation.value.length, 0).to.be.equal;
+    });
+
+    it('should return the correct value from getTaskText', function() {
+      const taskText = SurveyTask.getTaskText(task);
+      expect(taskText).to.be.a('string');
+    });
+
+    it('should correctly return a default task', function() {
```

Comment:
Default task should be an object. Optionally, you can also test the properties of the task object to make sure they're correct.